### PR TITLE
Add cleanup security scans 

### PR DESCRIPTION
**Pull request by m1kra**
Transition action "cleanup security scans" should be available for VirtualServer and CloudHost, RAS-1215.

### CODE REVIEW

File: src/ralph/virtual/models.py [hunk 1]
```diff
@@ -267,6 +268,18 @@ def ip_addresses(self, value):
             base_object=self, ipaddress__address__in=to_delete
         ).delete()
 
+    @classmethod
+    @transition_action(
+        verbose_name=_('Cleanup security scans'),
+    )
+    def cleanup_security_scans(cls, instances, **kwargs):
```
Comment (mkurek) [inline src/ralph/virtual/models.py]: the same method is defined already for `DataCenterAsset`. Please extract it to some common class or put somewhere as plain function - `transition_action` has additional argument - `models` - see https://github.com/allegro/ralph/blob/ng/src/ralph/deployment/deployment.py for examples.

**Comment by vi4m** [comment]
@m1kra description is required for all pull requests (can re-paste from RAS-1215)

**Comment by m1kra** [comment]
@vi4m Thank you, I have added the description.

@mkurek I have moved `cleanup_security_scans` to a new file in security app and added config loading it in `AppConfig.ready()`, following the approach in the deployment module. 
I am not quite sure if security module is the best location for it, but I find this solution logical. I was also thinking of adding it to transition app or base assets, so the additional config wouldn't be necessary.

**Review by mkurek — DISMISSED** [review]

File: src/ralph/security/apps.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+from ralph.apps import RalphAppConfig
+
+
+class SecurityConfig(RalphAppConfig):
+    name = 'ralph.security'
+    verbose_name = 'Security'
+
+    def get_load_modules_when_ready(self):
+        """
+        security/transitions.py will be loaded alongside modules reurned by
```
Comment (ar4s) [inline src/ralph/security/apps.py]: reurned -> returned

**Review by ar4s — CHANGES_REQUESTED** [review]

File: src/ralph/security/apps.py (hunk 2 above)
Comment (m1kra) [reply src/ralph/security/apps.py]: Done.

**Review by mkurek — APPROVED** [review]

**Review by ar4s — APPROVED** [review]

**Pull request merged**